benefits/core/migrations/0002_paymentprocessor_audience_paymentprocessor_client_id_and_more.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 5.0.2 on 2024-02-28 23:16
+
+import benefits.core.models
+import benefits.secrets
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("core", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="paymentprocessor",
+            name="audience",
+            field=models.TextField(default="audience"),
+            preserve_default=False,
```

Comment:
When generating this migration file, Django forced me to provide a default value for `audience` and `client_id` since the fields do not allow null.

We can use the admin interface to update the values.